### DISCUSSION TITLE
PDF Cover image display title

### DISCUSSION THREAD

**Discussion by gitgilleszeimet** [thegetty/quire#445] [Get Help] [answered]
I have been experimenting with a Quire default project and I replaced the default cover image with a photo. If I do so, the title is no longer getting displayed on the cover of the PDF.

Am I doing something wrong, or is that simply how Quire works?

Thank you for your help,

Gilles

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @gitgilleszeimet. Just to clarify, is the photo that you replaced the default cover image with appearing on the website version of your project?

**Reply by gitgilleszeimet** [reply]
Hi @Erin-Cecele. I created a new Quire project for testing purposes. In order to familiarise myself with he tool, I deleted the content of the downloads folder and recreated the documents (PDF, epub, etc.). As expected, the cover of the newly creted PDF had the spiral pattern in the background with the title of the publication in a rectangular box on top. Then I replaced the spiral with a photo (duo.jpg) and created a new PDF just to see what will happen. Now the cover of the PDF only featured the photo, but the title no longer got displayed.

**Reply by geealbers** [reply]
@gitgilleszeimet, that's a strange bug I hadn't seen before, but am seeing it on our end now too. Adding the following CSS to the `static/css/custom.css` file in the project fixed it for me. Does it work for you?

```css
.quire-cover__hero .hero-body .container {
  z-index: 1;
}
```

**Comment by gitgilleszeimet** [comment]
Hi @geealbers, thank you for your help. Your solution solved the problem.
